### ISSUE TITLE
All folders in workspace opens

### ISSUE THREAD

**Issue by gunjan1909** [microsoft/vscode#136152]
Issue Type: <b>Bug</b>

After updating from windows 10 to windows 11, each time I open Vs Code, all my folders in workspace are open/completely expanded, I have to close/ unexpand them all manually.
Example: If I have 7 folders in my workspace and each folder have different projects, then all my folders and project are expanded in the explorer

VS Code version: Code 1.61.2 (6cba118ac49a1b88332f312a8f67186f7f3c1643, 2021-10-19T14:57:20.575Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz (8 x 2400)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.84GB (1.11GB free)|
|Process Argv|--crash-reporter-id 54b87bfe-60a6-4d42-bc71-ceb591ca03f8|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.2.2
prettier-vscode|esb|9.0.0
auto-close-tag|for|0.5.13
code-runner|for|0.11.6
copilot|Git|1.6.3571
vscode-pull-request-github|Git|0.31.1
stylelint-plus|hex|0.56.6
vscode-gutter-preview|kis|0.28.0
dotenv|mik|1.0.1
python|ms-|2021.10.1365161279
vscode-pylance|ms-|2021.10.3
jupyter|ms-|2021.9.1101343141
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.3
cpptools|ms-|1.7.1
vsliveshare|ms-|1.0.5065
LiveServer|rit|5.6.1
JavaScriptSnippets|xab|1.8.0

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
pygetstartedt3:30385195
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedc:30364665
pythonvspyt187:30373474
vsqsis200cf:30386380
vsaa593:30376534
vssld246cf:30386378

```

</details>

**Comment by gjsjohnmurray** [comment]
/duplicate of #134665 and fixed in 1.62 which is scheduled to release before the end of next week.

**Comment by gunjan1909** [comment]
Thanks
